pom.xml

```diff
@@ -39,7 +39,34 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
-	</dependencies>
+		<dependency>
+			<groupId>org.springframework.data</groupId>
+			<artifactId>spring-data-commons</artifactId>
+		</dependency>
```

Comment:
Worin unterscheidet sich diese Dependency von spring-boot-starter-data-jpa ? Brauchen wir beide?

Reply:
Wenn ichs richtig verstehe, ist spring-data-commons in spring-boot-starter-data-jpa enthalten, lul. Hab das hier entfernt: [82a20eb](https://github.com/BaranselKarakas/character-sheet-creator/pull/14/commits/82a20eb197b1b771b5d1a9b9c0368131bb7904b5)